tests/test.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+include('../../include/cli_check.php');
+include('./slowlog_functions.php');
+
+global $reserved_words, $debug;
+
+$debug = true;
+
+//get_table_associations(1, 40069);
+//get_table_associations(1, 34832);
+//get_table_associations(7, 274637);
+get_table_associations(9, 545976);
```

Comment:
Hardcoded test?

Reply:
Yea, is very specific to the slowlog you are looking at though. At some point, if we can get people to volunteer a few large slow logs, we could memorialize the tests.